### PR TITLE
FIO. Retry signed trx.

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

**Comment by andreyvEze** [comment]
approved, waiting for package.json updates for FIOSDK to merge

**Comment by andreyvEze** [comment]
Do not merge for now, we are doing more testing

File: src/fio/fioEngine.js [hunk 1]
```diff
@@ -622,13 +623,110 @@ export class FioEngine extends CurrencyEngine {
...
+              shuffleArray(
+                this.currencyInfo.defaultSettings.apiUrls.map(apiUrl => () =>
+                  this.fioRetryApiRequest(apiUrl, e.requestParams)
+                )
+              )
+            )
+            break
+          } else if (
+            e.errorCode &&
+            [800, 801].indexOf(parseInt(e.errorCode)) > -1
```
Comment (andreyvEze) [inline src/fio/fioEngine.js:714]: add const for error codes

**Comment by andreyvEze** [comment]
approved with Eliran

**Review by paullinator — APPROVED** [review]

**Comment by paullinator** [comment]
Approved by @thehobbit85

**Pull request merged**